api/models/user.js

```diff
@@ -3,7 +3,7 @@ const mongoose = require("mongoose");
 const UserSchema = new mongoose.Schema({
   email: { type: String, required: true },
   password: { type: String, required: true },
-  userName: { type: String, required: true},
+  userName: { type: String, required: true, default: "no username"},
```

Comment:
is the default value here really necessary? if this is a required field, the user will have to put something in the form, therefore the default will never actually be activated. if the default value gets put in our database it means our forms / front end checks arent working as they are supposed to

Reply:
True, I guess it's only if any of us have old records in our local database without a username it would break, but we can just delete those records ourselves